lambda/templates/ecr_scan_notifications_lambda.json.tpl

```diff
@@ -0,0 +1,24 @@
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Effect": "Allow",
+      "Action": [
+        "logs:CreateLogStream",
+        "logs:PutLogEvents"
+      ],
+      "Resource": [
+        "arn:aws:logs:eu-west-2:${account_id}:log-group:/aws/lambda/tdr-ecr-scan-notifications-mgmt",
+        "arn:aws:logs:eu-west-2:${account_id}:log-group:/aws/lambda/tdr-ecr-scan-notifications-mgmt:log-stream:*"
+      ]
+    },
+    {
+      "Effect": "Allow",
+      "Action": "ses:SendEmail",
+      "Resource": [
+        "arn:aws:ses:eu-west-2:${account_id}:identity/aws_tdr_management@nationalarchives.gov.uk",
```

Comment:
Is there a way to send in the resource arns as variables? Would make it a bit more generic?

Reply:
It would be better but this is for a specific lambda so it's not too bad. I tried to pass in the arn but terraform doesn't have any data sources for SES and the actual SES module is defined in the tdr-aws-accounts repo so I can't reference it in the tdr-terraform-backend root file either. I think we'll have to leave it.

Reply:
Fair enough